docs/source/guides/libraries/swift_ios.rst

```diff
@@ -0,0 +1,282 @@
+      # use this command for device instead:
+      # cp lib/* iroha-ios/libs/iOS/
+      cp lib/* iroha-ios/libs/Simulator/
+
+18.	Don't forget to copy generated headers:
+
+  .. code-block:: bash
+
+      cp -a include/. iroha-ios/headers/
+
+19.	Now it's time to manually config Xcode project for the sample application. Open SwiftyIroha.xcodeproj:
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_001.png
+ 
+20.	Select SwiftyIrohaExample.xcodeproj general tab and link SwiftProtobuf framework from iroha-ios/SwiftProtobuf.framework location
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_002.png
+
+21.	Select SwiftGRPC.xcodeproj project and remove zlib-example target from it:
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_003.png
+ 
+22.	Go to Proto group and remove it (In future this step will be removed, but for now it's needed for sample app to be built):
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_004.png
+
+23.	Congratulations! We are done. Select SwiftyIrohaExample target, choose iPhone simulator device and build the application to make sure we've done everything correctly:
+
+.. image:: https://github.com/hyperledger/iroha/raw/develop/docs/image_assets/iroha_swift_guide/iroha_swift_guide_005.png
+
+Before we launch the application and test it we should deploy Iroha on our local machine and launch it.
+
+There is good news - steps 1-18 shouldn't be done manually every time - here is the script which does it automatically.
+
+The script for iOS client installation and setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+iroha_preparation_script.sh
+"""""""""""""""""""""""""""
+
+  .. code-block:: bash
+
+    #!/bin/bash
+
+    #download ios client and update dependencies
+    git clone https://github.com/soramitsu/iroha-ios.git
+    cd iroha-ios/
+    carthage update --platform iOS
+    cd SwiftyIrohaExample
+    carthage update --platform iOS
+
+    #build grpc client for sample application
+    cd grpc-swift/
+    rm -rf ./*
+    rm -rf ./.*
+    git clone --branch 0.3.3 https://github.com/grpc/grpc-swift.git .
+    make
+
+    #back to the root where script was executed
+    cd ../../..
+
+    #download and build Iroha library for iOS
+    curl https://raw.githubusercontent.com/hyperledger/iroha/master/shared_model/packages/ios/ios-build.sh > ios-build.sh
+
+    #optional step - sometimes connection timeout appears when using git: scheme instead of https url
+    sed -i '' 's|git://github.com/hyperledger/iroha-ed25519|https://github.com/hyperledger/iroha-ed25519.git|g' ios-build.sh
+
+    #build library
+    chmod +x ios-build.sh
+    ./ios-build.sh SIMULATOR64 Debug
+
+    #place artefacts to proper sample's locations
```

Comment:
typo: art**e**facts -> art**i**facts

take a look at the similar below

Reply:
will do